.github/workflows/api-php-static-code-check.yml

```diff
@@ -10,14 +10,19 @@ jobs:
         php-versions: [ '8.0' ]
         nextcloud: [ 'stable23' ]
         database: [ 'sqlite' ]
+        include:
+          - php-versions: 8.1
+            nextcloud: pre-release
+            database: sqlite
+            experimental: true
     name: "phpstan: Nextcloud ${{ matrix.nextcloud }}"
     steps:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Set up php
         uses: shivammathur/setup-php@master
         with:
-          php-version: 7.4
+          php-version: ${{ matrix.php-versions }}
```

Comment:
We have to change the configuration to check at php 7.4 level then

Reply:
Done :)